resources/public/openapi/spec.yml

```diff
@@ -480,6 +503,22 @@ components:
       items:
         $ref: '#/components/schemas/Competition'
 
+    CompetitionAdmin:
```

Comment:
I think it might be me that's confused, but shouldn't CompetitionAdmin also contain a competition? Or is that unnecessary information? Because I guess that id it the id of the user? Or what id is it? Maybe add a description of the id as well?

Reply:
The ID belongs to the competition admin yes, and is from the start a user ID. However, it this point I do not see a reason to have the ID of the competition represented in the object as only the get for a specific ID of the competition exists. Therefore the resource you get already has a specific ID in the path of the request.

Depending on how the creation of admins are made, this object should probably be extended with a property called *competition*. If it should probably be a `writeOnly` property. However, I have no thought this through yet. At this point I do not see a reason to have a competition ID here.

Reply:
Write only property sound like a good idea and sure it can wait. 
I was just getting confused about what the id stood for, maybe a clarification is needed?

Reply:
Some descriptions would probably be nice!